BitwardenShared/UI/Platform/Application/Views/PasswordStrengthIndicator.swift

```diff
@@ -55,9 +58,11 @@ struct PasswordStrengthIndicator: View {
 
                 Spacer()
 
-                Text(passwordStrength.text ?? "")
-                    .foregroundColor(Color(asset: passwordStrength.color))
-                    .styleGuide(.footnote)
+                if passwordTextCount >= requiredTextCount {
```

Comment:
⛏️ Very minor but should we be using a computed property here `hasPasswordMinimumLength`? So it can be reused for the `fillWidth` as well.